docs/modules/user-guide/partials/con_using-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using top-level attribute
+
+[role="_abstract"]
+
+This section explains how to use top-level attribute in devfiles in order to reuse global attributes in different sections of the same devfile.
+
+.Devfile variable
+
+The devfile 2.0 specification includes global variables, which defines key-value pairs.
+
+```yaml
+variables:
+  my-var: "some value"
+```
+
+These variables can be referenced in non-key string fields within the devfile using `{{my-var}}`. It is used to template different versions of a devfile by setting only the variable field.
+
+.Top-level attribute
+
+Using top-level attribute, you can define attributes as a map of implementation-dependent free-form YAML attributes on the top level in a devfile. You can use `{{attribute-name}}`` to refer to the attribute with the key `attribute-name`. To resolve attributes within a devfile only the top-level attributes are available. Attributes defined on other levels, like under a command, cannot be resolved.
+
+For example, we have defined a global attribute Java version number on the top level and referenced the value of the variable in the container image field:
```

Comment:
You want to avoid words like `we, our, us` Also, the phrasing is a little hard to understand. What do you think of this rephrase: 

`See the following example of top-level attributes.`